internal/fsapi/syscall_errno.go

```diff
@@ -0,0 +1,98 @@
+package fsapi
+
+import "syscall"
+
+func syscallToErrno(errno syscall.Errno) Errno {
```

Comment:
in the future, we can make a build constraint on this for GOOS that don't have syscall.Errno. I really don't know which that is.